src/App.svelte

```diff
@@ -29,16 +30,18 @@
   };
 </script>
 
-<div
+<main
   class="flex w-full flex-1 overflow-y-auto overflow-x-hidden bg-white text-gray-900 dark:bg-gray-900 dark:text-white"
 >
-  <Sidebar />
   <div class="flex flex-1 flex-col overflow-hidden">
+    <Sidebar />
     <TopNav />
-    <Router {routes} />
+    <div on:click={hideSidebar} class="h-screen lg:ml-56 p-5">
```

Comment:
Can use `flex-1` here since parent container is a flexbox. also removing extra padding since it breaks the toolbar in the file view
also added `overflow-hidden` so the child views can easily use `overflow-auto` to add scrollbars
```suggestion
    <div on:click={hideSidebar} class="flex-1 lg:ml-56 flex flex-col overflow-hidden">
```

Reply:
Updated